man/man8/zfs-send.8

```diff
@@ -470,7 +473,7 @@ To receive an incremental send from the original snapshot to something
 redacted with respect to a subset of the set of snapshots the initial snapshot
 was redacted with respect to.
 In this case, each block that was redacted in the original is still redacted
-(redacting with respect to additional snapshots causes less data to be redacted
+(redactng with respect to additional snapshots causes less data to be redacted
```

Comment:
Stray typo here.

Reply:
I'm pretty confused how that happened - it's not like it's near anything I was editing. Whoops.